src/inmanta/db/versions/v202504220.py

```diff
@@ -0,0 +1,29 @@
+"""
+Copyright 2025 Inmanta
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Contact: code@inmanta.com
+"""
+
+from asyncpg import Connection
+
+
+async def update(connection: Connection) -> None:
+    """
+    Rename some rps columns and values
+    """
+    schema = """
+        ALTER TABLE public.resource_persistent_state ADD COLUMN is_deploying BOOLEAN DEFAULT FALSE;
```

Comment:
We probably need an index involving this field, but I'm not sure if now's the time (i.e. if we already know enough about how it will be used). But I think we can't go wrong with simply an index on `environment, is_deploying`. @wouterdb wdyt?

Reply:
Why do we need an index?